sahi/slicing.py

```diff
@@ -317,7 +317,7 @@ def _export_single_slice(image: np.ndarray, output_dir: str, slice_file_name: st
         image_pil = read_image_as_pil(image)
         slice_file_path = str(Path(output_dir) / slice_file_name)
         # export sliced image
-        image_pil.save(slice_file_path)
+        image_pil.save(slice_file_path, quality="keep")
```

Comment:
FYI using `quality="keep"` fails (silently) for jpg ...

Reply:
FYI @fcakyon - I'd raise an issue but these don't seem to be available on this repo. I'd probably try/except the code executed in the thread pool too, just so we don't silently fail again.

Reply:
Hi, thanks for the hint. What is the error message? Can you try with following line?

`image_pil.save(slice_file_path, 'JPEG', quality="keep")`

Reply:
Sorry, I just deleted the quality parameter to make it work = ) It looks like it won't work as this still errors for me.

```python
from PIL import Image
Image.new("RGB", (100, 100)).save("tmp.jpg", "JPEG", quality="keep")
```

Reply:
Oh from above:

> According to the pillow documentation: "The value keep is only valid for JPEG files and will retain the original image quality level, subsampling, and qtables."

The problem is these aren't JPG files, these are slices of numpy arrays or similar, so I don't think anything will be kept. Maybe just revert to the quality argument you had?